pages/settings/monetization.vue

```diff
@@ -213,6 +214,20 @@ export default defineNuxtComponent({
       }
       stopLoading()
     },
+    getAccountInputAutocompletion(selectedWallet, accountType) {
+      if (selectedWallet === 'paypal') {
+        return accountType === 'email' ? 'email' : 'tel'
+      } else {
+        switch (accountType) {
+          case 'email':
+            return 'email'
+          case 'phone':
+            return 'tel'
+          case 'user_handle':
+            return 'username'
+        }
+      }
+    },
```

Comment:
this should use computed properties, not a method

Reply:
It's fixed ^^.